Sources/Serveable.swift

```diff
@@ -0,0 +1,10 @@
+
+/// Serves your order to a local or a remote bar
+
+public protocol Serveable {
+
+    associatedtype S
+
+    func serve(order: Order, delivery: (Delivery<S>)->())
```

Comment:
Wasn't the syntax we were going to use:

```
Bar.serve(order) { result in
    ...
}
```

And an order was `Serveable`, not the service.